library/Cube/CubeRenderer/ServiceStatusCubeRenderer.php

```diff
@@ -48,6 +48,10 @@ public function renderFacts($facts)
         $main = '';
         $sub = '';
         foreach ($parts as $class => $count) {
+            if ($count === 0) {
```

Comment:
Why?

Reply:
If the Problems only control is enabled, it may cause the OK|UP counter to be 0, we don't need to render it.